proto/attestation/verification.proto

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright 2023 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package oak.attestation.v1;
+
+option go_package = "proto/oak/attestation/v1";
+option java_multiple_files = true;
+option java_package = "com.google.oak.attestation.v1";
+
+// Represents a verification result. Can be extended to return certain
+// measurements and other detail to the client for further processing.
+message VerificationResult {
+  // Indicates whether or not the verification passed.
+  bool success = 1;
```

Comment:
Perhaps this may be an enum from the start, so we can add more values if needed in the future for specific error situations. For now it may just be unspecified / success / generic_failure.

Reply:
Done.